core/isolate.rs

```diff
@@ -689,7 +1415,7 @@ impl Future for Isolate {
 /// IsolateHandle is a thread safe handle on an Isolate. It exposed thread safe V8 functions.
 #[derive(Clone)]
 pub struct IsolateHandle {
-  shared_libdeno_isolate: Arc<Mutex<Option<*mut libdeno::DenoIsolate>>>,
+  shared_isolate: Arc<Mutex<Option<Isolate>>>,
 }
```

Comment:
I think we can remove this struct altogether - it was exposing handle to libdeno isolate which is no longer a thing. Usage should be changed to shared a handle to isolate behind a mutex.

Reply:
Okay it's useful to stop isolate from another thread. No idea how to implement it though...